leapp.spec

```diff
@@ -46,16 +46,20 @@ Requires:   libvirt-python
 Requires:   nmap
 Requires:   sshpass
 Requires:   python-enum34
+Requires:   docker 
+Requires:   rsync
```

Comment:
We do depend on `rsync` for the CLI to work (might be one more reason for the P2P rsync use case, but there migth be another things to solve in that regard)